Documentation/Hosting/AmazonGameLift/GameLiftLocal.md

```diff
@@ -0,0 +1,53 @@
+# MultiplayerSample Project with Amazon GameLift Local
+
+This README covers testing and running MultiplayerSample with Amazon GameLift Local.
+
+## Local testing with [GameLiftLocal](https://docs.aws.amazon.com/gamelift/latest/developerguide/integration-testing-local.html)
+
+### Setup
+
+1. Install the [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html), if you don't already have it. You will need it to create a game session with GameLiftLocal.
+1. Download Java Runtime Environment: https://www.java.com/en/download/manual.jsp
+1. Download the [AmazonGameLift Server SDK](https://gamelift-release.s3-us-west-2.amazonaws.com/GameLift_06_03_2021.zip) and extract the "GameLiftLocal-1.0.5" directory somewhere easy to find.
+1. Enable the "AWSGameLift" and "MPSGameLift" gem by adding them to MultiplayerSample/Gem/Code/enabled_gems.cmake
```

Comment:
```suggestion
1. Enable the "AWSGameLift" and "MPSGameLift" gems by adding them to MultiplayerSample/Gem/Code/enabled_gems.cmake
```

Reply:
This step is incorrect.
Th enable_gems.cmake has been deprecated and even it wasn't not deprecated, the enable_gems.cmake shouldn't be modified directly.

Instead the `o3de.py` CLI script `enable-gem command should be used to enable the GameLift gem.

```sh
PS> scripts\o3de.bat enable-gem -pp <path-to-multiplayer-sample> -gn MPSGameLift
```

Reply:
Getting an error using o3de.bat ...
Not sure where o3de.bat got the path C:\O3DE\0.0.0.0 from... that's not a directory.
Investigating...

Reply:
Fixed. I removed a bad path in o3de manifest. Not sure why this particular command needed it, but all is well.

Reply:
All is well, project manager doesn't update the gem toggles when opening the "Configure Gems". It must be closed and reopened entirely.

Reply:
I believe @AMZN-alexpete knows about the issue with the Project Manager needing to be restart to update the enabled gems model in the UI.

It makes sense though as the model is completely in memory.
A solution going forward might be to use a FileWatcher as we do for the Asset Processor.
We could then monitor when either the `project.json` or the `engine.json` file changes on disk and refresh the Gem model

Reply:
Doc updated to include commanline and project manager options for adding gem

Reply:
@AMZN-Gene while the Gem Catalog in Project Manager doesn't currently watch the `project.json` or `engine.json` files for changes, you should be able to use the refresh button on the Gem Catalog page (in the header or the hamburger menu) to cause it to reload the gems and not have to restart the Project Manager.  Another, less obvious way is to open another project's Gem Catalog, then return to the current project's Gem Catalog.  We don't currently reload the Gem Catalog every time it is opened if the project remains the same to cut down on load time.

Can you create a feature request for us to add `QFileSystemWatcher` on `project.json`, `engine.json` and `o3de_manifest.json` (maybe gem.json too) in the Gem Catalog to detect these major changes?  We currently do monitor `project.json` files on the Projects Screen, but not the Gem Catalog.